lib/cmd/fh3/notifications/list.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('List notifications of a RHMAP App'),
+  'examples' : [{
+    cmd : 'fhc notifications list --app=<app> --env=<environment> --audit=<true>',
+    desc : i18n._('List notifications of the <app>')}],
+  'demand' : ['app', 'env'],
+  'alias' : {
+    'app':'a',
+    'env':'e',
+    'audit':'au',
+    'json': 'j',
+    0 : 'app',
+    1 : 'env',
+    2 : 'audit',
+    3 : 'json'
+  },
+  'describe' : {
+    'app'   : i18n._("Unique 24 character GUID of your cloud application."),
+    'env'   : i18n._("Unique 24 character GUID of the environment where this cloud application is deployed."),
+    'audit' : i18n._("Flag to list the notifications"),
+    'json' : i18n._("Output in json format")
+  },
+  'customCmd': function(params, cb) {
+    var uri = "box/srv/1.1/app/eventlog/listEvents";
+    var payload = {appGuid: params.app, eventGroup: "NOTIFICATION", env: params.env};
```

Comment:
Notifications list can be very long (and usually is). I will also check in ngui (we have pagination there) if there is api option that allows us to get latest x notifications (which is usually something that we want to do).
Printing really long list of notifications when most recent ones are on the top may be a problem for this command. This seems to be outside target of this ticket, but I think it's best to also improve commands as we move. Consider this change optional.